docs/docs/glossary.md

```diff
@@ -1,8 +1,13 @@
----
-title: Glossary
----
+import HorizontalNavList from "../../www/src/components/horizontal-nav-list.js"
 
-When you're new to Gatsby there can be a lot of words that seem alien. This glossary aims to give you a 10,000ft overview of common terms and what they mean to the layperson.
+# Glossary
+
+When you're new to Gatsby there can be a lot of words to learn. This glossary aims to give you a 10,000-foot overview of common terms and what they mean for Gatsby sites.
+
+<HorizontalNavList
+items={"ABCDEFGHIJKLMNOPQRSTUVWXYZ".split("")}
```

Comment:
So awesome! and you could pass an array for more flexibility later?

Reply:
Yeah that was the idea! And it's a lot shorter than a big array or object of links and titles.